src/storekit.cs

```diff
@@ -58,19 +58,23 @@ partial interface SKDownload {
 		[Export ("state")]
 		SKDownloadState State { get; }
 #if MONOMAC
```

Comment:
Why not removing the MONOMAC?

Reply:
I don't this one can be removed since the #if and the #else contain the same method names

Reply:
If they have the same method names, you need to merge the definitions so that they have the same availability attributes.